### ISSUE TITLE
Ignore routing settings at the start of routing

### ISSUE THREAD

**Issue by Jean-BaptisteC** [organicmaps/organicmaps#9622]
When you are on dirt road with unpaved road routing settings enabled, calculating routing fail.
I suggest to add a distance tolerance (500m or 1km) to allow to calculate routing even if routing settings is enabled.

**Comment by Misalf-git** [comment]
I guess the (unpaved) beginning part of a potential route could have any length.

**Comment by Admetny1** [comment]
Yes, it's very annoying issue. Also, it's not only about the start of routing but the end as well. In the sample attached there is an unpaved road between the current location and the blue spot on route.

![Image](https://github.com/user-attachments/assets/3ecaf049-f9d5-4c7a-87ca-b409c9a29a32)
